src/track-labs-container.ts

```diff
@@ -0,0 +1,31 @@
+import { log } from '@guardian/libs';
+import { EventTimer } from './event-timer';
+
+/**
+ * Collect commercial metrics on:
+ * - whether the page contains a Guardian Labs container element (aka 'dumathoin'), and if so
+ * - when the element is scrolled into view
+ */
+const initTrackLabsContainer = () => {
+	const target = document.querySelector('section.dumathoin');
+	if (target === null) return;
+
+	const eventTimer = EventTimer.get();
+
+	log('commercial', 'Page has labs container');
+	eventTimer.trigger('hasLabsContainer');
```

Comment:
Non-blocking: it might be worth considering using `event.Timer.setProperty` and treating `hasLabsContainer` and possibly `labsContainerInView` as properties of the page rather than as metrics. 
`labsContainerInView` seems like it could be considered an "event" and that we might be interested in analyzing the timestamps associated with it. I'm not sure that the same is true of `hasLabsContainer` though.